earth_enterprise/rpms/opengee-server/post-install.sh

```diff
@@ -48,7 +47,7 @@ main_postinstall()
     setup_geserver_daemon
 
     # 5) Register publish root
-    $BASEINSTALLDIR_OPT/bin/geconfigurepublishroot --noprompt --path=$PUBLISHER_ROOT
+    "$BASEINSTALLDIR_OPT/bin/geconfigurepublishroot" --noprompt "--path=$PUBLISHER_ROOT"
```

Comment:
Minor: would it make more sense to have `--path="$PUBLISHER_ROOT"` rather than `"--path=$PUBLISHER_ROOT"`?  I moved the open quote.

Reply:
That depends on your style.  I'm fine with it either way.